usaspending_api/disaster/management/sql/disaster_covid19_file_d2_awards.sql

```diff
@@ -64,9 +64,7 @@ SELECT
     "transaction_fabs"."place_of_performance_zip4a" AS "primary_place_of_performance_zip_4",
     "transaction_fabs"."place_of_performance_congr" AS "primary_place_of_performance_congressional_district",
     "transaction_fabs"."place_of_performance_forei" AS "primary_place_of_performance_foreign_location",
-    "transaction_fabs"."cfda_number" AS "cfda_number",
-    "transaction_fabs"."cfda_title" AS "cfda_title",
-    "transaction_fabs"."assistance_type" AS "assistance_type_code",
+    (SELECT STRING_AGG(DISTINCT CONCAT(U0."cfda_number", ': ', U0."cfda_title"), '; ' ) AS "total" FROM "transaction_fabs" U0 INNER JOIN "transaction_normalized" U1 ON (U0."transaction_id" = U1."id") WHERE U1."award_id" = ("vw_award_search"."award_id") GROUP BY U1."award_id") AS "cfda_numbers_and_titles",    "transaction_fabs"."assistance_type" AS "assistance_type_code",
```

Comment:
`"transaction_fabs"."assistance_type" AS "assistance_type_code",` should be on a different line.

Also, `vw_award_search` is never referenced in a `from` so this is blowing up the query. Did you means `"awards"."id`?

Reply:
should be fixed